recipes/xpdconf/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "xpdConf" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "7f83a8221c711a2896f61e457b11add176b9288330cb44da5bb540fb14254696" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/xpdAcq/{{ name }}/releases/download/{{ version }}/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - databroker
+    - pyyaml
+
+test:
+  imports:
+    - xpdconf
+
+about:
+  home: http://github.com/xpdAcq/xpdConf
+  license: BSD-3-Clause 
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Configuration for XPD beamlines'
+
+extra:
+  recipe-maintainers:
+    - CJ-Wright
+    - chiahaoliu
```

Comment:
@chiahaoliu are you ok with being a maintainer?

Reply:
yes